lib/jhove_service.rb

```diff
@@ -74,7 +74,8 @@ def exec_command(command)
   # @param output_file [Pathname,String] the output file to write the XML to, defaults to filename specified in jhove_output
   # @return [String] The jhove-toolkit command to be exectuted in a system call
   def get_jhove_command(content,output_file = jhove_output)
-    args = "-h xml -o \"#{output_file}\" \\\"#{content}"
```

Comment:
Not really related to your PR, but would it be possible to rename the `content` argument to something a little more descriptive such as `input_path` or similar?